### PR TITLE
typo FR correction

### PR DESCRIPTION
**Pull request by Davidffry**

### CODE REVIEW

**Comment by shanemcd** [comment]
@rooftopcellist Is it ok to change this file directly or do we need to update these in whatever translation app we're using these days?

**Comment by chris93111** [comment]
Hello anyone can merge this ? in 21.7.0 is always present

File: awx/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -722,7 +722,7 @@ msgstr "DTSTART valide obligatoire dans rrule. La valeur doit commencer par : DT
 #: awx/api/serializers.py:4657
 msgid ""
 "DTSTART cannot be a naive datetime.  Specify ;TZINFO= or YYYYMMDDTHHMMSSZZ."
-msgstr "DTSTART ne peut correspondre à une DateHeure naïve. Spécifier ;TZINFO= ou YYYYMMDDTHHMMSSZZ."
+msgstr "DTSTART ne peut correspondre à une simple datetime. Spécifier ;TZINFO= ou YYYYMMDDTHHMMSSZZ."
```
Comment (rooftopcellist) [inline awx/locale/fr/LC_MESSAGES/django.po:725]: Is this supposed to be English?

**Comment by rooftopcellist** [comment]
I have requested that our translators review this PR and incorporate these changes into the translation memory in Memsource so that they will be present then next time we pull newly translated strings and commit them to repo (next week Oct 24th).

**Comment by croe22** [comment]
Date-heure naïve or DateHeure naïve are both correct translatios of naive datetime.
I looked at automation-controller-aap-2.3-1 in Memsource, which containes 2 files, and the only string containing 'datetime' was the following:
DTSTART cannot be a naive datetime.  Specify ;TZINFO= or YYYYMMDDTHHMMSSZZ. 
Now translated as:
DTSTART ne peut correspondre à une date-heure naïve. Spécifier ;TZINFO= ou YYYYMMDDTHHMMSSZZ.

**Comment by akus062381** [comment]
Closing- see @rooftopcellist as to the reason why

**Pull request closed without merging**